install.sh

```diff
@@ -330,9 +330,9 @@ nvm_do_install() {
   BASH_OR_ZSH=false
 
   if [ -z "${NVM_PROFILE-}" ] ; then
-    echo "=> Profile not found. Tried ${NVM_PROFILE} (as defined in \$PROFILE), ~/.bashrc, ~/.bash_profile, ~/.zshrc, and ~/.profile."
```

Comment:
hmm, couldn't `$PROFILE` define a file that doesn't exist?

Reply:
I just move it to the end of the sentence, didn't remove it, would that be good?

The commit message was wrong, the point here is the installation script use not defined `${NVM_PROFILE}` :)

Reply:
Totally, but it seems like `NVM_PROFILE` will only be undefined some of the time; which means that some of the time, displaying it is very helpful.

Reply:
hmmmm ... I don't understand, there is a line `if [ -z "${NVM_PROFILE-}" ] ; then` before, doesn't that mean it's always not defined in this code block?

Reply:
Right - `$NVM_PROFILE` will of course always be empty in this block; but the purpose of the message is to indicate the exact value the user had defined in their profile file. `nvm_detect_profile` looks at `$PROFILE` - I assume that's the original purpose of this message?

Reply:
I'm not very sure, so is this message going to tell the users it has tried `${NVM_PROFILE}` this variable? Or just tell them that it has tried the following files?

Reply:
I think that *if* a profile file defines `$PROFILE`, but the `-f` check fails, then the error message should display the value of `$PROFILE` and indicate that it doesn't exist.